src/detector.c

```diff
@@ -354,11 +356,11 @@ void train_detector(char *datacfg, char *cfgfile, char *weightfile, int *gpus, i
             iter_map = iteration;
             mean_average_precision = validate_detector_map(datacfg, cfgfile, weightfile, 0.25, 0.5, 0, net.letter_box, &net_map);// &net_combined);
             printf("\n mean_average_precision (mAP@0.5) = %f \n", mean_average_precision);
-            if (mean_average_precision > best_map) {
+            if (mean_average_precision > 80.0) {
```

Comment:
Doesn't this occur at every iteration? If our trained model has over 80 precision, does this mean it will save at every iteration?

Reply:
correct, it creates a lot of weight files but we need the weights file for more than 80 percent since the FE can test various weight files for optimizing their situation.